test/inductor/test_torchinductor.py

```diff
@@ -10558,6 +10559,99 @@ def fn(x):
         x = torch.rand([4, 4, 3], dtype=torch.float64)
         self.common(fn, (x,))
 
+    @skipCUDAIf(not SM80OrLater, "uses bfloat16 which requires SM >= 80")
+    @fresh_inductor_cache()
+    # We only support dtypeview for abi_conpatible aoti
+    @torch._inductor.config.patch(abi_compatible=True)
+    def test_dtypeview(self):
+        # https://github.com/pytorch/pytorch/issues/126338
+        def fn(x, y, x_dtype, x2):
+            x = x.view(x_dtype)
+            y = y.view(x_dtype) + 1
+            x2 = x2.view(x_dtype) + 1
+            return x @ y, x2 @ x
+
+        def _get_primitive_bitwidth(dtype):
+            if dtype.is_floating_point:
+                return torch.finfo(dtype).bits
+            else:
+                return torch.iinfo(dtype).bits
+
+        test_dtypes = [
+            torch.float32,
+            torch.float64,
+            torch.float16,
+            torch.bfloat16,
+            torch.uint8,
+            torch.int8,
+            torch.int16,
+            torch.int32,
+            torch.int64,
+        ]
+        for test_dtype_x in test_dtypes:
+            for test_dtype_y in test_dtypes:
+                # @ operation needs arguments to be the same dtype
+                for view_dtype in test_dtypes:
+                    if (
+                        _get_primitive_bitwidth(test_dtype_x)
+                        != _get_primitive_bitwidth(test_dtype_y)
+                        or _get_primitive_bitwidth(test_dtype_x)
+                        != _get_primitive_bitwidth(view_dtype)
+                        or view_dtype
+                        in [
+                            torch.int32,
+                            torch.int64,
+                            torch.int16,
+                            torch.uint8,
+                            torch.int8,
+                        ]
```

Comment:
just directly return `x, y, x2` and dont have skipped dtypes

Reply:
fixed in https://github.com/pytorch/pytorch/pull/128883/commits/dfd1b9f77cda724c0a7a9124a3a249562fde50af